app/components/splash/index.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * Create Splash Page
+ * by id.ly Team on 4/14/2018
+ */
+
+//Import Libraries
+import React, {Component} from 'react';
+import { Text, View, StyleSheet,
+         Animated, Easing, Image
+        } from 'react-native';
+import {bindActionCreators} from 'redux';
+import {connect} from 'react-redux';
+import * as ReduxActions from '../../actions';
+import {Actions} from 'react-native-router-flux';
+import styles from './styles';
+
+class Splash extends Component {
+    static navigationOptions = {
+        header: null
+    }
+
+    componentWillMount() {
+        setTimeout(() => {
+            this.props.navigation.navigate('home');
```

Comment:
@vanctran when you finish researching navigation techniques can you explain the differences between using .navigate vs. actions.home()?